SimplyPluralApiBodies.yml

```diff
@@ -13,20 +13,6 @@ paths:
 
 components:
   schemas:
-    commons:
```

Comment:
Why are these removed? 
These are in use by for example writing:

 "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/color/description"